specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/stable/2016-05-01/Backup.json

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
Could you remove all files under stable/2016-05-01? This API version has been deprecated.